### ISSUE TITLE
Local changes detected

### ISSUE THREAD

**Issue by tsimonq2** [Ferdi265/wl-mirror-debian#2]
```
dpkg-source: info: local changes detected, the modified files are:
 wl-mirror-debian/Dockerfile
 wl-mirror-debian/README.md
 wl-mirror-debian/build.sh
 wl-mirror-debian/docker.sh
 wl-mirror-debian/lint.sh
```

These differ from what's in the tarball provided by `uscan --download-current-release`. You can either make those completely match upstream, or just remove them from the tarball when you make it. Your call, I'm not particular; I've just removed all of these files from the Debian copy for now.

**Comment by Ferdi265** [comment]
Thanks for the heads-up! I'm not very familiar with the exact flow of the debian packaging tools yet, so I wasn't sure how things were supposed to look exactly, but I will take a look at it and fix this by the next release.

**Comment by Ferdi265** [comment]
I'm not really able to reproduce that error message from dpkg-source, so my workflow definitely differs from the debian workflow, or at least your workflow.

How exactly do you build the package? (i.e., in which directory do you run `debuild` or `dpkg-source`, starting from a clean clone of this repo) I only use the `debian/` folder in this repo as a template and copy it into the source tar balls from the upstream `wl-mirror` project, though I guess that's not the standard way to do this. Do you extract the upstream tarballs inside this repo and then build the package in-tree?

The structure of my current build scripts is very ad-hoc and feels incorrect, and I definitely want to change this package to be more "correct".

**Comment by tsimonq2** [comment]
If you're looking for the absolute, most correct way, I'd recommend [git-buildpackage](https://wiki.debian.org/PackagingWithGit).

If you don't want to track the upstream source in two places, just track your `debian/` directory and nothing else (but keep that top-level `debian/` directory). That's how we do it in Lubuntu. :wink:

**Comment by Ferdi265** [comment]
I have removed all extra files from the repo, this repo is now just the bare `debian/` directory, since I don't want to track upstream source twice.

I have checked and the package now builds fine with:

```sh
origtargz -u
gbp buildpackage \
    --git-debian-branch=main \
    --git-ignore-new \
    --git-builder="debuild -i -I -us -uc"
```

I have also released 0.16.0 in the meantime.

For completeness, my personal build scripts are now maintained separately in [debian-docker-buildscripts](https://github.com/Ferdi265/debian-docker-buildscripts).
